.pre-commit-hooks.yaml

```diff
@@ -65,3 +65,9 @@
     entry: "\uFFFD"
     language: pygrep
     types: [text]
+-   id: rst-directive-colons
+    name: rst directives end with two colons
+    description: 'Detect mistake of rst directive not ending with double colon'
+    entry: '^\s*\.\. [a-z]+(:|:{3,}|:\s:)$'
```

Comment:
why not `:\s\s:`? if we're going to add more beyond the base case probably use something like `:(?!:).*$`